iib/workers/tasks/build_merge_index_image.py

```diff
@@ -251,17 +200,21 @@ def handle_merge_request(
 
         set_request_state(request_id, 'in_progress', 'Deprecating bundles in the deprecation list')
         log.info('Deprecating bundles in the deprecation list')
-        intermediate_bundles = source_index_bundles + missing_bundles
-        deprecate_bundles = _get_bundles_from_deprecation_list(
+        intermediate_bundles = [
+            bundle['bundlePath']
+            for bundles in (source_index_bundles, missing_bundles)
```

Comment:
is this from another open PR?

Reply:
Nope, I changed the input for `get_bundles_from_deprecation_list` hence this change. the other PR has the change in `_add_bundles_missing_in_source`

Reply:
I'll change it to use `chain` though